rules-reviewed/jboss-eap/websphere/websphere-xml.windup.xml

```diff
@@ -75,6 +75,9 @@
                     </hint>
                 </iteration>
             </perform>
+            <where param="file_ext">
+                <matches pattern="xml|xmi"/>
```

Comment:
That is mentioned at hhttp://www-01.ibm.com/support/knowledgecenter/SS7K4U_6.1.0/com.ibm.websphere.zseries.doc/info/zseries/ae/crun_app_bindings.html?cp=SS7K4U_6.1.0%2F1-7-7-2-0 and I found more hits on ibm's websphere forum